test/e2e/copy_from_tar_test.go

```diff
@@ -151,4 +151,31 @@ func TestCopyTarSrc(t *testing.T) {
 
 		imgpkg.Run([]string{"copy", "--tar", tempBundleTarFile, "--to-repo", fakeRegistry.ReferenceOnTestServer("copied-bundle")})
 	})
+
+	t.Run("When there is no longer access to the origin registry the copy is successful", func(t *testing.T) {
+		env := helpers.BuildEnv(t)
+		imgpkg := helpers.Imgpkg{T: t, L: helpers.Logger{}, ImgpkgPath: env.ImgpkgPath}
+		defer env.Cleanup()
+
+		destinationFakeRegistry := helpers.NewFakeRegistry(t, &helpers.Logger{LogLevel: helpers.LogDebug})
+		defer destinationFakeRegistry.CleanUp()
+
+		var tempBundleTarFile string
+		logger.Section("create tar from bundle", func() {
+			originFakeRegistry := helpers.NewFakeRegistry(t, &helpers.Logger{LogLevel: helpers.LogDebug})
```

Comment:
can we have an assertion on FakeRegistry checking that it was not accessed at all.

Reply:
I can add a comment in the next line, but the CleanUp stops the server when this `logger.Section` ends. So the server will not be on.